addons/html_builder/static/src/builder/options/alert_option.js

```diff
@@ -1,7 +1,31 @@
+import { fonts } from "@html_editor/utils/fonts";
 import { registry } from "@web/core/registry";
 
 registry.category("sidebar-element-option").add("AlertOption", {
     template: "html_builder.AlertOption",
     selector: ".s_alert",
     sequence: 5,
 });
+
+registry.category("website-builder-actions").add("alertIcon", {
+    apply: ({ editingElement, param: className }) => {
```

Comment:
To be discussed but intuitively, I think I would have done something like:
```.js
    apply: ({ editingElement, param: className }) => {
        const icon = editingElement.querySelector(".s_alert_icon");
        if (!icon) {
            return;
        }
        fonts.computeFonts();
        const allFaIcons = fonts.fontIcons[0].alias;
        icon.classList.remove(...allFaIcons);
        icon.classList.add(className);
    },
    clean: ({ editingElement, param: className }) => {
        const icon = editingElement.querySelector(".s_alert_icon");
        icon.classList.remove(className);
    },
```

So that in the `clean`, the widget only cleans the class for which its responsible and in the apply, it makes sure that the other classes (added by another widget) are removed before its own class can be applied. But again, we can discuss it ;)